internal/controller/snapshot/snapshot_adapter.go

```diff
@@ -785,6 +785,8 @@ func (a *Adapter) createIntegrationPipelineRun(application *applicationapiv1alph
 	if err != nil {
 		return nil, fmt.Errorf("failed to set snapshot %s as ControllerReference of pipelineRun: %w", snapshot.Name, err)
 	}
+
+	a.logger.Info(fmt.Sprintf("pipelineRun data: %+v", pipelineRun))
```

Comment:
Is this log entry needed? Just want to make sure we don't flood the service logs if it isn't.

Reply:
No, thanks